formhero/providers/dbform/tests.py

```diff
@@ -3,31 +3,44 @@
 from formhero.apps.models import App
 from formhero.forms.models import Form
 from formhero.providers.dbform import Backend
-from formhero.providers.dbform.models import FormEntry
+from formhero.providers.dbform.models import FormSubmission
 
 
 class DatabaseFormTest(TestCase):
-
     def setUp(self):
+        """
+        Create an app, a form, data, and a backend for the tests.
+        Variable db_config is empty as we don't need any
+        configurations (but django complains if nothing is given
+        to the config field of Form)
+        """
         self.an_app = App.objects.create(name='this_app')
         self.data = {'name': 'Dave',
-                'inquirer email': 'dave@example.com',
-                'message': 'This is a test message'
-                }
-        self.a_form = Form.objects.create(app=self.an_app, name='Form name', handler='db', config=self.data)
+                     'inquirer email': 'dave@example.com',
+                     'message': 'This is a test message'
+                     }
+        self.db_config = ''
+        self.a_form = Form.objects.create(app=self.an_app, name='Form name', handler='db', config=self.db_config)
```

Comment:
Was the empty string due to a complaint? Because we shouldn't have to supply a value.

If it's complaining add a default clause to the json field.